src/core/app_backend.tf

```diff
@@ -1012,3 +1012,44 @@ module "app_backend_web_test_api" {
   ]
 
 }
+
+# -----------------------------------------------
+# Alerts
+# -----------------------------------------------
+
+data "azurerm_app_service" "app_backend_app_services" {
+  for_each            = toset(var.app_backend_names)
+  name                = format("%s-app-${each.value}", local.project)
+  resource_group_name = azurerm_resource_group.rg_linux.name
+}
+
+resource "azurerm_monitor_metric_alert" "too_many_error" {
```

Comment:
can you find a better name for: "too_many_error"

Reply:
`too_may_http_5xx` is better?

Reply:
Done in [fix after review](https://github.com/pagopa/io-infra/pull/296/commits/d04ab9cea130bcf3dcbebbf4d7a819d4ab95b1d1)